product-catalog/app/productcatalog/models/RichShippingRate.java

```diff
@@ -0,0 +1,15 @@
+package productcatalog.models;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.shippingmethods.ShippingRate;
+
+public class RichShippingRate extends Base {
```

Comment:
:moneybag::moneybag::moneybag:ShippingRate

@lauraluiz Do we have some naming conventions? ShippingMethodData?